modules/bookmark.js

```diff
@@ -0,0 +1,31 @@
+const Augur = require("augurbot"),
+  u = require("../utils/utils");
+
+// Message context menu for bookmarking a message.
+
+const Module = new Augur.Module()
+.addInteractionCommand({name: "Bookmark",
+  commandId: undefined,
+  process: async (interaction) => {
+    try {
+      await interaction?.deferReply({ephemeral: true});
```

Comment:
Are you planning to use this command via a `MessageComponent` at some point? If not, this line can be removed as `deferReply()` is only available for `MessageComponentInteraction`s.

```suggestion
```

Reply:
Interestingly, the D.js guide shows `.deferReply()` on all interactions. (It would make sense for it to exist on `CommandInteraction` and `ContextMenuInteraction` as well.) It's missing from the docs, though. My guess is that it exists, but docs are incomplete. Should have been `await interaction.deferReply?.()` to properly indicate that was there as a failsafe in case it didn't exist.

Reply:
Confirmed `.deferReply()` is an undocumented method. Corrected the chaining.